Orchestrator.py

```diff
@@ -569,9 +570,11 @@ def orchestrate(self, action ):
 				self.sequenceTiers(Orchestrator.TIER_STOP)
 
 				for currTier in self.sequencedTiersList:
-				
+					logger.info("IgnoreStopValue for {} tier is {}".format(currTier, self.tierSpecDict[currTier][Orchestrator.TIER_STOP][Orchestrator.TIER_IGNORE_STOP]))
```

Comment:
Please change this message to logger.**debug**, since you have a logger.**info** log message when it has been confirmed that IgnoreStop is set to true